docs/examples/unit_tests/test_cases/python3/setup/executeTest.sh

```diff
@@ -1,44 +1,45 @@
-
+#!/bin/bash
 ############
 # Author: TSRK Prasad
 # Date: 18-Sep-2016
 #
 # script fragment used by ../../execute.sh to perform run-time tests. This script is not invoked directly
 #
-# variables manipulated:
```

Comment:
Date: 28-Sep-2017
Previous Versions: 18-Sep-2016

Please make the above change in all the scripts